rollup/presets/package-typescript.js

```diff
@@ -1,16 +1,21 @@
 import babel from '@rollup/plugin-babel';
 import typescript from '@rollup/plugin-typescript';
 import { terser } from 'rollup-plugin-terser';
+import path from 'path';
 import { packageBabelPreset } from '../configs/babel-presets';
 import { externalsForPlugin } from '../configs/externals';
 
-export function packageTypescript() {
-	return [
-		{
-			input: 'src/index.ts',
+export function packageTypescript(pkg) {
```

Comment:
@romainmenke let me know your thoughts about this change. This moves the configuration decisions to `exports` which feel more sensible. I haven't tested out of this nor consolidated the JS counterpart but wanted to check if this worked and how did you feel about it.

Reply:
Tests already suggest I need to look into this further